src/www/CoreServer.cpp

```diff
@@ -46,4 +46,14 @@ void CoreServer::init(void) throw(std::exception) {
 
 const std::vector<VirtualServer>& CoreServer::getVirtualServers(void) const { return m_virtual_servers; }
 
+std::vector<VirtualServer>& CoreServer::getVirtualServers(void) { return m_virtual_servers; }
+
+bool CoreServer::addVirtualServer() {
+  VirtualServer server("Peter", 8080);
+  m_virtual_servers.push_back(server);
+  return true;
+}
+
+bool CoreServer::removeVirtualServer() { return true; }
```

Comment:
Maybe call this function "true" so we can program more in a Haskell style☠️😂

Reply:
Hell no. (: